rotkehlchen/api/v1/schemas.py

```diff
@@ -705,10 +704,26 @@ def make_history_event_filter(
         else:
             filter_query = HistoryEventFilterQuery.make(**common_arguments)
 
-        return {
+        base_filter = {
             'filter_query': filter_query,
-            'group_by_event_ids': data['group_by_event_ids'],
         }
+        base_filter.update(self.get_extra_fields(data))
+        return base_filter
+
+    def make_extra_common_arguments(self, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
```suggestion
    def make_extra_filtering_arguments(self, data: dict[str, Any]) -> dict[str, Any]:
```

a name that suggests where this will be used is better. `common` doesn't really means anything here or add value to get a quick idea for where this will be used